src/Interfaces/LazySet.jl

```diff
@@ -878,3 +879,93 @@ true
 ```
 """
 isconvextype(X::Type{<:LazySet}) = false
+
+"""
+    area(X::LazySet{N}) where {N}
+
+Compute the area of a set.
+
+### Input
+
+- `X` -- set
+
+### Output
+
+A number representing the area of `X`.
+"""
+function area(X::LazySet{N}) where {N}
+    if dim(X) == 2
+        return area_2D(X)
+    else
+        throw(ArgumentError("the area function only applies to two-dimensional sets, " *
+                            "but the given set is $(dim(X))-dimensional"))
+    end
+end
+
+"""
+    area_2D(X::LazySet{N}) where {N}
+
+Compute the area of a two-dimensional polytopic set using the Shoelace formula.
+
+### Input
+
+- `X` -- two-dimensional set
+
+### Output
+
+A number representing the area of `X`.
+
+### Algorithm
+
+Let `m` be the number of vertices of `X`. The following instances are special-cased:
+
+- `m = 0, 1, 2`: the output is zero.
+- `m = 3`: the triangle case is computed using the Shoelace formula with 3 points.
+- `m = 4`: the quadrilateral case is obtained by the factored version of the Shoelace
+           formula with 4 points.
+
+Otherwise, the general Shoelace formula is used; for detals see the wikipedia article
+[Shoelace formula](https://en.wikipedia.org/wiki/Shoelace_formula).
+"""
+function area_2D(X::LazySet{N}) where {N}
+    @assert dim(X) == 2 "this function only applies to two-dimensional sets, " *
+    "but the given set is $(dim(X))-dimensional"
+
+    vlist = vertices_list(X)
+    m = length(vlist)
+
+    if 0 <= m <= 2
+        return zero(N)
+    end
+
+    if m == 3 # triangle
+        A = vlist[1]
+        B = vlist[2]
+        C = vlist[3]
+        res = A[1] * (B[2] - C[2]) + B[1] * (C[2] - A[2]) + C[1] * (A[2] - B[2])
+        return abs(res/2)
```

Comment:
It is probably beneficial if you outsource these cases to methods as well because then it can dispatch on the vector type.

Reply:
ok